README.md

```diff
@@ -2,4 +2,586 @@
 
 Load bundles of translated strings.
 
+## Installation
 
+```
+npm install ilib-resbundle
+
+or
+
+yarn add ilib-resbundle
+```
+
+## Quick Start
+
+In `myapp/src/example.js` file:
+
+```javascript
+import ResBundle from 'ilib-resbundle';
+
+[...]
+    const rb = new ResBundle({
+        name: "strings",
+        locale: appLocale
+    });
+
+    const buttonLabel = rb.getString("Send now");
+
+    const dialogBodyStr = rb.getString("one#There is {count} invitation|#There are {count} notifications");
+    const dialogBody = dialogBodyStr.formatChoice(count);
+```
+
+If your app's locale was, let's say, German, then in `myapp/locale/de/strings.json` file:
+
+```
+{
+    "Send now": "Jetzt schicken",
+    ""one#There is {count} invitation|#There are {count} invitations": "one#Es gibt {count} Einladung|#Es gibt {count} Einladungen"
+}
+```
+
+In your `myapp/package.json` file (only the relevant parts are shown):
+
+```
+{
+    "dependencies": {
+        "loctool": "^2.18.0"
+    },
+    "scripts": {
+        "localize": "loctool"     <- most of the settings come from the project.json (see below)
+    }
+}
+```
+
+In your `myapp/project.json` file, which configures the loctool (only the relevant parts are shown):
+
+```
+{
+    "name": "myapp",
+    "plugins": [ "ilib-loctool-javascript" ],
+    "resourceDirs": {
+        "javascript": "locale"
+    },
+    "resourceFileTypes": {
+        "javascript":"ilib-loctool-javascript-resource"
+    },
+    "settings": {
+    "xliffsDir": "./xliffs",
```

Comment:
I think it's better to give an indentation to `xliffsDir`

Reply:
Done